nav2_gps_waypoint_follower_demo/launch/gazebo_gps_world.launch.py

```diff
@@ -1,5 +1,5 @@
-# Copyright (c) 2018 Intel Corporation
-#
+# Copyright (C) 2023 Open Source Robotics Foundation
```

Comment:
Why OSRF here?

Reply:
done in https://github.com/ros-navigation/navigation2_tutorials/pull/115/commits/114b265fb9394a7185a0e76dfce590795dfdb9e0